{"status":"OK","result":{"originalLocale":"ru","allowViewHistory":false,"creationTimeSeconds":1401922974,"rating":11,"authorHandle":"alaudo","modificationTimeSeconds":1401945678,"id":12552,"title":"\u003cp\u003eCode execution error on a lengthy input -- how to debug this issue?\u003c/p\u003e","locale":"en","content":"\u003cdiv class\u003d\"ttypography\"\u003e\u003cp\u003eHi\u003c/p\u003e\u003cp\u003eI have deliberately read the rules and completely understand the reasoning of site\u0027s administration behind the decision not to disclose test sets. And I agree to put up with this decision for this place\u0027s sake, for this is the only place where I can peek at the solutions submitted by the leaders and compare them to my owns.\u003c/p\u003e\u003cp\u003eBut what should I do if -- in my opinion -- my code seems to be correct but it just fails on a certain lengthy test?\u003c/p\u003e\u003cp\u003eIf the code just returns a wrong answer, the system provides you with both expected and actual answers and one can use this information for further debugging.\u003c/p\u003e\u003cp\u003eBut how can I tackle execution errors in my code? I have seemingly working code that dies on one of lengthy tests: \u003ca href\u003d\"/contest/439/submission/6810527\" title\u003d\"Submission 6810527 by alaudo\"\u003e6810527\u003c/a\u003e. And I am completely perplexed what to do now: does it fail while reading the data input, during computation or output?\u003c/p\u003e\u003cp\u003eI fully agree with people agrumenting that it is my fault and that there are multiple submissions where this doesn\u0027t happen. But I want to understand why my code is failing!\u003c/p\u003e\u003cp\u003eIs there any possibility to return any kind of debugging information on execution failure? I can, if required, to output some information to stderr so I can see it later along with test results (or only 200 characters to avoid passing the input data through into stderr).\u003c/p\u003e\u003cp\u003eThanks in advance,\u003c/p\u003e\u003ch2\u003eAlexander.\u003c/h2\u003e\u003cp\u003e\u003cstrong\u003ePS:\u003c/strong\u003e As I was prompted in the comments, I can basically output any textual informaiton into stdout and look at it in the results of my code execution. So, just wrapping my complete code into a single global try/catch already did the trick and I found out that it was an overflow execption in LINQ Sum() extension method...\u003c/p\u003e\u003c/div\u003e","tags":["test","debug","output"]}}